cscs-checks/system/openstack/s3_check.py

```diff
@@ -0,0 +1,132 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class S3apiCheck(rfm.RunOnlyRegressionTest):
+    def __init__(self):
+        super().__init__()
+        endpoint = 'object.cscs.ch'
+        self.descr = 'S3API check for (%s)' % endpoint
+        self.tags = {'ops', endpoint}
```

Comment:
We should also add another tag here, e.g., `object_storage`? Regarding putting `endpoint` as a tag, I would  only do it if I wanted to select a test by  `endpoint`.

Reply:
Set 'object_store' as tag value. For the moment we have only one production object store endpoint.